### DISCUSSION TITLE
Thoughts from the Creator

### DISCUSSION THREAD

**Discussion by 3xp11i** [prakhartiwari0/my-web-shortcuts#12] [Announcements]
This project is currently in its initial phase, one of the first contributions (#7 #8) has been made :zap:, and its **Open Source journey has been started**. I got a lot of [appreciation](https://x.com/Prakhartiwari0/status/1712577234987368666?s=20) from a lot of amazing folks in the tech community. And I am thankful to those who are eager to contribute to its development.:pray:

I created the very basic & buggy version, which lacks a lot of fundamental features, in under a week. Then launched a [20-minute long video](https://www.youtube.com/watch?v=x_fYfporwmA) explaining what this project is about and how can people contribute to it. Because [Hacktoberfest](https://hacktoberfest.com) is going on, contributors are excited to find and contribute to good Open Source Projects. And I'm lucky to have friends in tech who too want to improve this project. :smile: 

I'll be making this project more contributor-friendly, and create quality documentation :spiral_notepad: & content for both users and contributors. Guessing from the rate of contributions, I am hopeful that the first version with all the fundamental features will be ready by the first week of November. Therefore I am planning to **officially launch it on the 7th day of November this year**. But there is a caveat, the duration of [Chrome Extension review process](https://developer.chrome.com/docs/webstore/review-process) varies. According to me, this extension is not very complex and doesn't require any dangerous permissions, therefore I think that it will not take a long time, but who knows? The launch can be delayed depending on the review duration. :hourglass: 

> *Whatever happens, what matters is that something new is created, something that adds value to people's lives, a new idea, new opportunities and new conversations. **Collaboration and value addition should be our highest priority in Open Source**.*

With these lovely words, I will end this announcement, and **wish everyone a Happy Hacktoberfest :fire:, and Happy Contributing :heart:.** 

_Take care of yourselves :relaxed:_

**Comment by sayande717** [comment]
This reminds me of the fact that JavaScript was also made within 10 days (I think it's true?!), and it also had (or has xd) a lot of issues. But people still use it because it gets the job done.

This was the way i thought about this when you'd released v1.0.0 and it had that major bug (closed issue #44). But everything worked out in the end! That's really all that matters. It's not about what you ship on day 1, it's about how quickly you squash all bugs, which, to a degree, shows your dedication to making it work.

Hats off for making all this happen and solving a real issue people actually face :)

**Reply by 3xp11i** [reply]
Thanks for your kind words Sayan!
